src/Aspire.Hosting.PostgreSQL/PostgresContainerImageTags.cs

```diff
@@ -11,4 +11,7 @@ internal static class PostgresContainerImageTags
     public const string PgAdminRegistry = "docker.io";
     public const string PgAdminImage = "dpage/pgadmin4";
     public const string PgAdminTag = "8.8";
+    public const string PgwebRegistry = "docker.io";
```

Comment:
`Pgweb` or `PgWeb`?

Reply:
`pgweb` is the correct one, but we could use `PgWeb`.